woocommerce/woocommerce-ios
# [Tablet Orders] Order list UI updates

Partially closes #12149

## Description
This PR addresses most of the UI issues from the design review regarding the Order list. We have left outside of this PR those changes related to semantic colours, since all might need to be reviewed as per p1709628907203209/1708465836.162389-slack-C0354HSNUJH . This will be prioritized and tackled separately.

## Changes
* Adds a divider line to the top of the `OrderListTableViewController`

<img width="664" alt="Screenshot 2024-02-29 at 19 19 18" src="https://github.com/woocommerce/woocommerce-ios/assets/3812076/b8a048e1-0d39-4347-8d56-07b59007ad17">

* Adds divider lines to the selected cell in the order list, it would seem these were overwritten by the background colour

<img width="638" alt="Screenshot 2024-02-29 at 19 22 42" src="https://github.com/woocommerce/woocommerce-ios/assets/3812076/c789416b-a6e3-482c-b492-690f3c0a0f51">

* Updates the Empty Orders screen with a new image (we do not tackle the background colour update, nor removing the navigation bar)

![simulator_screenshot_88F19ADC-1F69-4724-859F-D4606402A23D](https://github.com/woocommerce/woocommerce-ios/assets/3812076/b1ff343a-d6c8-4123-8be9-ab60efedfbb2)

## Testing instructions
* With `sideBySideViewForOrderForm` enabled, on a tablet or large iPhone.
* Go to Orders. Observe that the dividers appear in the Order list to the left, both at the top, as well as between all Order cells.
* On the same device, on a freshly created store (or a store with no orders), go to Orders. Observe the new "shopping bags" image and  the "No Order Selected" text below it. (PD: This is something we could improve in the future, the "Waiting for your first Order" text should be more prominent in this case)
